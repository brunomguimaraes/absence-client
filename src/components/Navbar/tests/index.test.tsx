import React from 'react';
import { shallow } from 'enzyme';

import NavBar from '..';

describe('Navbar', () => {
  it('renders the component', () => {
    const component = shallow(<NavBar />);
    // const calendarButton = component.find('#calendar')
    // const absenceButton = component.find('#calendar')
    
    expect(component).toMatchSnapshot();
  });
  
  // it('', () => {
  //   const calendarButton = component.find('#calendar')
  //   expect(calendarButton).
  // });

  // it('', () => {

  // });
});

