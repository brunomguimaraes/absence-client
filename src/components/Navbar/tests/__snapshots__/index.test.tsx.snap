// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders the component 1`] = `ShallowWrapper {}`;
